.gitignore

```diff
@@ -18,3 +18,4 @@ website/node_modules
 *.bak
 *~
 .*.swp
+config/lang/y.go
```

Comment:
This should not be in the `.gitignore`.  There's some oddness with go 1.4 and 1.5 but for now this file should just not be included in any changes, more ignored